src/runtimes/rust/builder.js

```diff
@@ -0,0 +1,49 @@
+const { readFile } = require('fs')
+const { basename, join } = require('path')
+const { promisify } = require('util')
+
+const pReadFile = promisify(readFile)
+
+const tmp = require('tmp-promise')
+const toml = require('toml')
+
+const { lstat } = require('../../utils/fs')
+const { runCommand } = require('../../utils/shell')
+
+const { BUILD_TARGET, MANIFEST_NAME } = require('./constants')
+
+const build = async ({ srcDir }) => {
```

Comment:
@mraerino @fasterthanlime Does this look okay to you from a Rust perspective? 🦀 

(You don't need to look at the whole PR, this file should contain most of the Rust-specific logic.)